src/handlers/RPCHelpers.cpp

```diff
@@ -1,5 +1,6 @@
-#include <backend/BackendInterface.h>
 #include <handlers/RPCHelpers.h>
+#include <handlers/Status.h>
```

Comment:
I think there are two files called RPCHelpers.cpp. Is that intentional?

Reply:
Um, i dont see two files, What are the relative paths to both of them?